pymc/tests/test_distributions_timeseries.py

```diff
@@ -12,20 +12,156 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+import aesara.tensor as at
 import numpy as np
 import pytest
 
+from scipy import stats
+
+import pymc as pm
+
 from pymc.aesaraf import floatX
 from pymc.distributions.continuous import Flat, Normal
-from pymc.distributions.timeseries import AR, AR1, GARCH11, EulerMaruyama
+from pymc.distributions.timeseries import (
+    AR,
+    AR1,
+    GARCH11,
+    EulerMaruyama,
+    GaussianRandomWalk,
+    gaussianrandomwalk,
+)
 from pymc.model import Model
 from pymc.sampling import sample, sample_posterior_predictive
 from pymc.tests.helpers import select_by_precision
 
-# pytestmark = pytest.mark.usefixtures("seeded_test")
-pytestmark = pytest.mark.xfail(reason="Timeseries not refactored")
+from .test_distributions_random import BaseTestDistributionRandom
```

Comment:
@ricardoV94 I moved the test class but i really dont think we should do this. i somewhat strongly feels test should be self contained in one module as it makes it easier to read and see whats going on

Reply:
I think the tests files should mirror the original codebase. So they should be in a file with the same name as where the functions/classes that it tests are defined. Ideally, timeseries tests should be in test/distributions/timeseries.py. This is how Aesara does it and its much easier to find what tests relate to what code, and also to spot gaps in our tests.

On the other hand, if you follow your logic you shouldn't have any test_timeseries.py at all. The random tests would go into test_distributions_random.py, the logp into test_distributions.py, and so on. Why would you have this test file at all?

Reply:
Unrelated to the discussion l, you should use absolute imports not relative one (that is the `.`)

Reply:
> On the other hand, if you follow your logic you shouldn't have any test_timeseries.py at all. The random tests would go into test_distributions_random.py, the logp into test_distributions.py, and so on. Why would you have this test file at all?

Yep agree, its already a bit confusing because we dont split per the convention of tests mirroring the codebase, we have two conventions going on at once. So if we go that way i think we should merge `test_distributions_random` and `test_distributions_logprob` etc but anyhow that is out of the scope of this pr :)

I'll update the relative import

Reply:
Opened up discussion in https://github.com/pymc-devs/pymc/discussions/5579

Reply:
Thanks